AK/ByteBuffer.h

```diff
@@ -92,16 +94,14 @@ class ByteBufferImpl : public RefCounted<ByteBufferImpl> {
 private:
     explicit ByteBufferImpl(size_t);
     ByteBufferImpl(const void*, size_t);
-    ByteBufferImpl() { }
 
-    u8* m_data { nullptr };
+    u8* m_data {};
```

Comment:
This looks weird. I mean, it probably does the right thing, but I wouldn't know that from looking at it.

Reply:
It is just initialization using uniform initialization syntax. This one could be changed back... No change here. I think I changed this because I was experimenting with something else and forgot to undo it.

Reply:
Reverted.